### PR TITLE
mods: 0.1.1 -> 0.2.0

### PR DESCRIPTION
**Pull request by mustafasegf**
Diff: https://github.com/charmbracelet/mods/compare/v0.1.1...v0.2.0
Changelog: https://github.com/charmbracelet/mods/releases/tag/v0.2.0

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Review by dit7ya — CHANGES_REQUESTED** [review]
Hi @mustafasegf, welcome to nixpkgs and thanks for your contribution.

Looking at the ofbord build failure logs, it seems like the package definition is missing some sort of `export HOME=$(mktemp -d)` in some step. There are many examples in nixpkgs of that. 

Let me know if you need more guidance.

**Comment by dit7ya** [comment]
Result of `nixpkgs-review pr 241415` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>mods</li>
  </ul>
</details>

File: pkgs/tools/misc/mods/default.nix [hunk 1]
```diff
@@ -29,6 +29,7 @@ buildGoModule rec {
 
     tests.version = testers.testVersion {
       package = mods;
+      command = "export HOME=$(mktemp -d); mods -v";
```
Comment (dit7ya) [inline pkgs/tools/misc/mods/default.nix:32]:
```suggestion
      command = "export HOME=$(mktemp -d); mods -v";
      command = "HOME=$(mktemp -d) mods -v";
```

**Review by dit7ya — APPROVED** [review]
LGTM apart from the minor style nit.

**Comment by mustafasegf** [comment]
@dit7ya i already change the style for test command

**Review by dit7ya — APPROVED** [review]

**Pull request merged**